docs/supported-libraries.md

```diff
@@ -17,111 +17,134 @@ or [contributing](../CONTRIBUTING.md).
-| [Spring Scheduling](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/scheduling/package-summary.html)   | 3.1+                          |
-| [Spring Web MVC](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/servlet/mvc/package-summary.html) | 3.1+                          |
-| [Spring Web Services](https://spring.io/projects/spring-ws)                                                                       | 2.0+                          |
-| [Spring WebFlux](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/reactive/package-summary.html)    | 5.0+                          |
-| [Spymemcached](https://github.com/couchbase/spymemcached)                                                                         | 2.12+                         |
-| [Tomcat JDBC Pool](https://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html)                                                       | 8.5.0+                        |
-| [Twilio](https://github.com/twilio/twilio-java)                                                                                   | 6.6+ (not including 8.x yet)  |
-| [Undertow](https://undertow.io/)                                                                                                  | 1.4+                          |
-| [Vaadin](https://vaadin.com/)                                                                                                     | 14.2+                         |
-| [Vert.x Web](https://vertx.io/docs/vertx-web/java/)                                                                               | 3.0+                          |
-| [Vert.x HttpClient](https://vertx.io/docs/apidocs/io/vertx/core/http/HttpClient.html)                                             | 3.0+                          |
-| [Vert.x Kafka Client](https://vertx.io/docs/vertx-kafka-client/java/)                                                             | 3.6+                          |
-| [Vert.x RxJava2](https://vertx.io/docs/vertx-rx/java2/)                                                                           | 3.5+                          |
-| [Vibur DBCP](https://www.vibur.org/)                                                                                              | 11.0+                         |
+| Library/Framework                                                                                                                           | Versions                      | Standalone Library Instrumentation [1]                                                                                                                                                                                                                                                                                                                                         | Semantic Conventions                                                                   |
```

Comment:
or somehow to mention that the versions listed are for auto-instrumentation (some library instrumentations only support newer versions)
```suggestion
| Library/Framework                                                                                                                           | Auto-instrumented versions                      | Standalone Library Instrumentation [1]                                                                                                                                                                                                                                                                                                                                         | Semantic Conventions                                                                   |
```